Userland/Utilities/js.cpp

```diff
@@ -600,6 +604,18 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
             s_editor->save_history(s_history_path.to_byte_string());
         });
 
+        s_editor->register_key_input_callback('C' & 0x3f, [](Line::Editor& editor) -> bool {
```

Comment:
In LibLine/Editor.cpp I do see usage of `register_key_input_callback(ctrl('C'), ...).` Could `ctrl()` be used here?

Reply:
It's not exported anywhere, but maybe it should be.
I'll patch it up later